rfcs/0102-moderation-team.md

```diff
@@ -0,0 +1,94 @@
+---
+feature: moderation team
+start-date: 2021-08-18
+author: tomberek
+co-authors: blaggacao
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: https://github.com/NixOS/rfcs/pull/98
+---
+
+# Summary
+[summary]: #summary
+
+Establish a team to perform moderation.
+
+# Motivation
+[motivation]: #motivation
+
+There is not currently any official mechanism for moderation action. It's not
+sustainable to have to call on Graham any time there's a spammer or conflict
+that requires moderation, and we'd like to help the community become more
+self-regulating.
+
+(adopted from #98)
+
+# Detailed design
+[design]: #detailed-design
+
+The Moderation Team is a volunteer group that may receive, invite, and evaluate
+applications to the team or alter the composition at any time with approval
+from the NixOS Foundation board or by any individual choosing to resign. The
+team's composition, contact information, and announcements should be maintained
+at
+[https://nixos.org/community/teams/moderation.html](https://nixos.org/community/teams/moderation.html).
+The team shall perform moderation activities on behalf of the NixOS Foundation
+for the discussions in the [official project
+spaces](https://nixos.org/community/index.html) as well as unofficial spaces
+that seek and reach such an agreement with the team. The team should utilize
+the [NixOS Foundation mission](https://nixos.org/community/index.html) and the
+following statement during their duties:
+
+```
+The NixOS Foundation aims to promote participation without regard to gender,
+sexual orientation, disability, ethnicity, age, or similar personal
+characteristics. We want to strive to create and foster community by providing
+an intentionally welcoming and safe environment where all feel valued and cared
+for, and where all are given opportunity to participate meaningfully. The
+Foundation will work with the community in service of this goal.
+```
+
+ref: [twitter](https://twitter.com/grhmc/status/1390775249424338944)
+
+# Examples and Interactions
+[examples-and-interactions]: #examples-and-interactions
+
+The initial Moderation Team is defined to be @grahamc, @zimbatm, @domenkozar,
+and @ryantm.
```

Comment:
Does anyone know if some people currently (actively) moderating any of the official spaces are missed? I cannot remember anyone else right now, I am probably missing someone.

Nominally Eelco Dolstra of course has moderation access on GitHub, but I am not sure when was the last time this was exercised.

Reply:
https://discourse.nixos.org/about

I think mainly I've been doing Discourse but in the cases of bigger issues, Mic92, Zimbatm, Rok, and Graham have helped.

Reply:
Thanks!